201-encrypt-create-new-vm-gallery-image/azuredeploy.json

```diff
@@ -52,31 +52,42 @@
                 "description": "Name of Subnet to which the VM NIC belongs to"
             }
         },
-        "AADClientID": {
+        "aadClientID": {
             "type": "string",
             "metadata": {
                 "description": "Client ID of AAD app which has permissions to KeyVault"
             }
         },
-        "AADClientSecret": {
+        "aadClientSecret": {
             "type": "securestring",
             "metadata": {
                 "description": "Client Secret of AAD app which has permissions to KeyVault"
             }
         },
-        "KeyVaultResourceID": {
+        "keyVaultResourceID": {
             "type": "string",
             "metadata": {
                 "description": "ResourceID of the KeyVault to place the volume encryption key"
             }
         },
-        "KeyVaultURL": {
+        "keyVaultURL": {
```

Comment:
Can you add to the readme how the customer needs to get this? Also what's the example for this